posix.mak

```diff
@@ -525,6 +525,17 @@ style: ../dscanner/dsc $(LIB)
 	@echo "Enforce space between binary operators"
 	grep -nrE "[[:alnum:]](==|!=|<=|<<|>>|>>>|^^)[[:alnum:]]|[[:alnum:]] (==|!=|<=|<<|>>|>>>|^^)[[:alnum:]]|[[:alnum:]](==|!=|<=|<<|>>|>>>|^^) [[:alnum:]]" $$(find . -name '*.d'); test $$? -eq 1
 
+	@echo "Validate changelog files (Do _not_ use REF in the title!)"
+	@echo " - Title line can't contain REF - it's already a link"
+	@echo " - After the title line an empty, separating line is expected"
+	@echo " - Which is supposed to be followed by a long description"
+	for file in $$(find changelog -name '*.dd') ; do  \
+		echo "Testing: $$file"; \
+		cat $$file | head -n1 | grep -nq '\$$' ; [ $$? -eq 0 ] && exit 1 ; \
```

Comment:
This just check for the dollar sign, right? I.e., it rejects all macros? This seems too restrictive. Macros that don't generate links should be allowed.